source/fuzz/transformation_duplicate_region_with_selection.cpp

```diff
@@ -499,8 +503,9 @@ void TransformationDuplicateRegionWithSelection::Apply(
       opt::Instruction::OperandList(
           {{SPV_OPERAND_TYPE_ID, {message_.merge_label_fresh_id()}}}));
   exit_block->AddInstruction(MakeUnique<opt::Instruction>(merge_branch_instr));
+  auto merge_branch_instr_cloned = merge_branch_instr.Clone(ir_context);
   duplicated_exit_block->AddInstruction(
-      MakeUnique<opt::Instruction>(merge_branch_instr));
+      std::unique_ptr<opt::Instruction>(merge_branch_instr_cloned));
```

Comment:
Could you avoid the "merge_branch_instr_cloned" temporary and just pass:

```
std::unique_ptr<opt::Instruction>(merge_branch_instr.Clone(ir_context)
```

as the parameter?